### ISSUE TITLE
TileDataset not working as expected

### ISSUE THREAD

**Issue by surya-narayanan** [Dana-Farber-AIOS/pathml#232]
**Describe the bug**
TileDataset object not working on multiplex images

**To Reproduce**
```
tile_dataset = TileDataset("./Scan1.h5")
tile_dataset.__len__() #returns 144
tile_dataset.__getitem__(0) #errors out
```

**Expected behavior**
tile_dataset.__getitem__(0) should return the first tile, without error

**Traceback**
```

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
/tmp/ipykernel_32507/806735975.py in <module>
----> 1 tile_dataset.__getitem__(0)

/tmp/ipykernel_32507/524695887.py in __getitem__(self, ix)
     70 
     71         # swap axes from HWC to CHW for pytorch
---> 72         im = tile_image.transpose(2, 0, 1)
     73 
     74         masks = np.stack(list(masks.values()), axis=0) if masks else None

ValueError: axes don't match array
```

**Extra context**
The Scan1 h5 file was created by using a .write() function on a slide, after running an empty pipeline on it.

**Comment by jacobrosenthaldfci** [comment]
thanks, fixed this in #233 and added tests

**Comment by surya-narayanan** [comment]
Thanks
